qiskit/circuit/library/standard_gates/rxx.py

```diff
@@ -111,6 +114,39 @@ def _define(self):
 
         self.definition = qc
 
+    def control(
+        self,
+        num_ctrl_qubits: int = 1,
+        label: str | None = None,
+        ctrl_state: str | int | None = None,
+        annotated: bool | None = None,
+    ):
+        """Return a (multi-)controlled-RXX gate.
+
+        Args:
+            num_ctrl_qubits: number of control qubits.
+            label: An optional label for the gate [Default: ``None``]
+            ctrl_state: control state expressed as integer,
+                string (e.g.``'110'``), or ``None``. If ``None``, use all 1s.
+            annotated: indicates whether the controlled gate can be implemented
+                as an annotated gate. If ``None``, this is set to ``True`` if
+                the gate contains free parameters, in which case it cannot
+                yet be synthesized.
```

Comment:
This formulation already works a bit better than the one where I gave suggestions, they probably should be unified (I don't care about the specific message, it's just that I think it should be explicit about when it's set to true or false).

Reply:
Updated to use this one consistently in 7a03c4f 👍🏻